### PR TITLE
Update window title

### PR DESCRIPTION
**Pull request by Dvinyanin**
Signed-off-by: Dvinyanin Alexandr <dvinyanin.alexandr@gmail.com>

# Contribution checklist

## Brief description
Screenshot from 2022-06-23 20-54-50

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues
https://front.hc.engineering/workbench%3Acomponent%3AWorkbenchApp/tracker%3Aapp%3ATracker/tracker%3Ateam%3ADefaultTeam/issues#tracker%3Acomponent%3AEditIssue%7C62b14f00319d758c9c6791df%7Ctracker%3Aclass%3AIssue%7Ccontent
A list of closed updated issues

### CODE REVIEW

**Comment by haiodo** [comment]
@Dvinyanin Let's also show workspace name in title.

File: models/view/src/index.ts [hunk 1]
```diff
@@ -158,6 +159,11 @@ export class TObjectFactory extends TClass implements ObjectFactory {
   component!: AnyComponent
 }
 
+@Mixin(view.mixin.ObjectTitle, core.class.Class)
+export class TObjectTitle extends TClass implements ObjectTitle {
+  getTitle!: Resource<<T extends Doc>(client: Client, ref: Ref<T>) => Promise<string>>
```
Comment (haiodo) [inline models/view/src/index.ts]: Better to name it titleProvider

File: packages/presentation/src/connect.ts [hunk 2]
```diff
@@ -69,7 +76,22 @@ export async function connect (title: string): Promise<Client | undefined> {
   }
 
   // Update window title
-  document.title = [fetchMetadataLocalStorage(login.metadata.CurrentWorkspace), title].filter((it) => it).join(' - ')
+  function setDocumentTitle (title?: string): void {
```
Comment (haiodo) [inline packages/presentation/src/connect.ts]: Le'ts move this code to Workbench plugin.

File: models/view/src/index.ts (hunk 1 above)
Comment (Dvinyanin) [reply models/view/src/index.ts]: Renamed

File: packages/presentation/src/connect.ts (hunk 2 above)
Comment (Dvinyanin) [reply packages/presentation/src/connect.ts]: Moved

**Review by haiodo — APPROVED** [review]

**Pull request merged**